azure-pipelines/build-release.yml

```diff
@@ -54,7 +54,7 @@ stages:
             Windows_Py_3_8:
               python.version: '3.8'
               vmImageName: windows-latest
-              uploadCoverage: "false"
+              uploadCoverage: "true"
```

Comment:
If this is what broke the CI result overview, we should reconsider making this change until that issue is resolved.

Reply:
Unfortunately then the coverage is reported as below 50%, as `mbed-devices` contained os specific code.

Reply:
So some tests were skipped on different platforms.

Reply:
OK, good not to hide this test coverage gap.